src/Components/CriticalCareRecording/Pain/CriticalCare__PainInputModal.res

```diff
@@ -55,13 +55,15 @@ let make = (
     }
   })
 
-  let getStatus = (min, minText, max, maxText, val) => {
-    switch (val >= min, val <= max) {
-    | (true, true) => ("Normal", "#059669")
-    | (true, false) => (maxText, "#DC2626")
-    | _ => (minText, "#DC2626")
-    }
+  let getLabel = (param: float) => {
+  let minText = "Low"
+  let maxText = "High"
+  switch param {
+  | 0.0 => (minText, "#059669")
+  | 5.0 => (maxText, "#DC2626")
+  | _ => ("", "")
```

Comment:
why not have a human-readable values for all the 5 cases like,
0 -> No pain (gray)
1 -> Noticeable (yellow)
2 -> Mild (orange)
3 -> High (light red)
4 -> Extremely high (red)
5 -> Unbearable (darker red)

Reply:
@khavinshankar should I change it then ?

Reply:
@khavinshankar Hey I clicked on the request for review by mistake

Reply:
Yes, please do

cc: @nihal467

Reply:
I have opened a new pr and done the changes as you described.please review it @khavinshankar